modules/microshift-cri-o-container-runtime.adoc

```diff
@@ -6,7 +6,7 @@
 [id="microshift-CRI-O-container-engine_{context}"]
 = Using a proxy in the CRI-O container runtime
 
-To use an HTTP(S) proxy in `CRI-O`, you must add a `Service` section to the configuration file and set the `HTTP_PROXY` and `HTTPS_PROXY` environment variables. You can also set the `NO_PROXY` variable to exclude a list of hosts from being proxied.
+To use a HTTP or HTTPS proxy in `CRI-O`, you must add a `Service` section to the configuration file and set the `HTTP_PROXY` and `HTTPS_PROXY` environment variables. You can also set the `NO_PROXY` variable to exclude a list of hosts from being proxied.
```

Comment:
This should be an "an" in front of HTTP or HTTP

There's more info on this topic in the IBM style guide here: https://www.ibm.com/docs/en/ibm-style?topic=grammar-articles#when-to-use-a-or-an

```suggestion
To use an HTTP or HTTPS proxy in `CRI-O`, you must add a `Service` section to the configuration file and set the `HTTP_PROXY` and `HTTPS_PROXY` environment variables. You can also set the `NO_PROXY` variable to exclude a list of hosts from being proxied.
```